src/FAudio_gstreamer.c

```diff
@@ -410,19 +458,74 @@ uint32_t FAudio_GSTREAMER_init(FAudioSourceVoice *pSourceVoice, uint32_t type)
 	 */
 	result = (FAudioGSTREAMER*) pSourceVoice->audio->pMalloc(sizeof(FAudioGSTREAMER));
 	FAudio_zero(result, sizeof(FAudioGSTREAMER));
+
 	result->pipeline = gst_pipeline_new(NULL);
-	g_assert(result->pipeline);
+	if (!result->pipeline)
+	{
+		LOG_ERROR(
+			pSourceVoice->audio,
+			"Unable to create gstreamer pipeline; is %u-bit gstreamer installed?",
+			sizeof(void *) * 8
+		)
+		goto free_without_bin;
+	}
+
 	decoder = gst_element_factory_make("decodebin", NULL);
-	g_assert(decoder);
+	if (!decoder)
```

Comment:
This however can fail if for some reason decodebin is not available, so correct to check here